### PR TITLE
Extend attestation type

### PR DESCRIPTION
**Pull request by motechFR**
### WHAT

Adding support for external credentials

### CODE REVIEW

File: src/prisma/schema.prisma [hunk 1]
```diff
@@ -2097,3 +2098,13 @@ model ReferralCodeUseEvent {
...
 }
+
+model ExternalProjects {
+  id            String  @id @default(uuid()) @db.Uuid
+  metadata      Json
+  recipient     String
+  schemaId      String?
+  // Fields specific to ceramic
+  ceramicId     String? @unique
+  ceramicRecord Json?
```
Comment (mattcasey) [inline src/prisma/schema.prisma:2109]: not sure this field is necessary. this table's purpose is just to preload projects

**Review by mattcasey — CHANGES_REQUESTED** [review]
Let's try to keep the external projects table separate from EAS Credentials. I can see why we'd save the ceramicId, so we can "issue" it to the user, but i don't think we need to store the entire JSON payload. We also have the full metadata, so seems like we could generate that credential if we wanted to

File: src/prisma/schema.prisma [hunk 2]
```diff
@@ -2098,3 +2099,13 @@ model ReferralCodeUseEvent {
   @@index([referrerId])
   @@index([refereeId])
 }
+
+model ExternalProjects {
```
Comment (mattcasey) [inline src/prisma/schema.prisma:2103]: this should be singular ExternalProject

File: src/prisma/schema.prisma [hunk 3]
```diff
@@ -2098,3 +2099,13 @@ model ReferralCodeUseEvent {
...
+  id        String  @id @default(uuid()) @db.Uuid
+  metadata  Json
+  recipient String
+  schemaId  String?
+  // Fields specific to ceramic
+  ceramicId String? @unique
+  source    String
```
Comment (motechFR) [inline src/prisma/schema.prisma:2110]: Suggestion; Add an EcosystemURL field, which is the @unique url for this project, and will also go into the final credential we create

Comment (mattcasey) [inline src/prisma/schema.prisma:2110]: maybe track projectId when we create a project from it?

Comment (valentinludu) [reply src/prisma/schema.prisma:2110]: done

File: src/prisma/schema.prisma (hunk 2 above)
Comment (valentinludu) [reply src/prisma/schema.prisma:2103]: done

**Review by mattcasey — APPROVED** [review]

**Pull request merged**